viam-orb-slam3.go

```diff
@@ -83,17 +82,14 @@ func init() {
 			return New(ctx, deps, c, logger, false, DefaultExecutableName)
 		},
 	})
-	config.RegisterServiceAttributeMapConverter(slam.Subtype, Model, func(attributes config.AttributeMap) (interface{}, error) {
-		var attrs slamConfig.AttrConfig
-		decoder, err := mapstructure.NewDecoder(&mapstructure.DecoderConfig{TagName: "json", Result: &attrs})
-		if err != nil {
-			return nil, err
-		}
-		if err := decoder.Decode(attributes); err != nil {
-			return nil, err
-		}
-		return &attrs, nil
-	}, &slamConfig.AttrConfig{})
+	config.RegisterServiceAttributeMapConverter(
+		slam.Subtype,
+		Model,
+		func(attributes config.AttributeMap) (interface{}, error) {
+			var conf slamConfig.AttrConfig
+			return config.TransformAttributeMapToStruct(&conf, attributes)
```

Comment:
Please also open a PR to remove `NewAttrConfig` from the slam repo as part of the ticket: https://viam.atlassian.net/browse/RSDK-2155

Reply:
done